lib/LaTeXML/MathParser.pm

```diff
@@ -714,7 +714,10 @@ sub node_to_lexeme {
           if (my $value = $$properties_pending{$attr}) {
             push @to_add, $value; } }
```

Comment:
Wouldn't it have been simpler to simply make `@to_add` a hash in the first place, and then `$to_add{$value}=1;` and use `keys %to_add` later ?

Reply:
sure, can do!